Sources/SwifterSwift/SceneKit/SCNVector3Extensions.swift

```diff
@@ -29,6 +29,14 @@ public extension SCNVector3 {
     var length: SceneKitFloat {
         return sqrt(pow(x, 2) + pow(y, 2) + pow(z, 2))
     }
+    
+    /// SwifterSwift: Returns the unit or normalized vector where `length = 1`.
+    ///
+    ///     SCNVector3(2, 3, 6).normal  -> SCNVector3(2/7, 3/7, 6/7)
+    ///
+    var normalized: SCNVector3 {
+        self / self.length
```

Comment:
Can you just inline the implementation here? Except for a couple of operators, we normally tend to not create an extension in terms of others, so it can be copied independently :)

Reply:
I see.. in that case I thinks it's easier and more readable to just use the SIMD3 again, no? .-.

Reply:
Sorry, I just meant instead of calling the `/` operator you created, call directly like:
```swift
var normalized: SCNVector3 {
        SCNVector3(vector.x / length, vector.y / length, vector.z / length)
}
```

Reply:
I think that is simpler than have to. use SIMD ... :)

Reply:
oh yea, sure..

Btw, funny story: I needed to adjust the test, bc I got an error:
> XCTAssertEqual failed: ("0.33333334") is not equal to ("0.3333333")

Don't think it's a big deal tho..

Reply:
good point, never actually used that..
But I think it's good as it is, since `4/6` is more accurate than `0.3333333`, right?